client/my-sites/sharing/connections/service.jsx

```diff
@@ -32,6 +33,8 @@ module.exports = React.createClass( {
 		onToggleSitewideConnection: React.PropTypes.func // Handler to invoke when toggling a connection to be shared sitewide
 	},
 
+	mixins: [ observe( 'connections' ) ],
```

Comment:
@aduth we had to add this in since fetches in ConnectionList were not triggering a re-render in this component. 

On a first page load with a clean redux store, this lead to a "stuck" loading state. Even though the data was available on the ConnectionList. 

Let us know if adding this would cause any issues. 😄